src/test/osd/osd-scrub-repair.sh

```diff
@@ -925,7 +927,7 @@ function TEST_corrupt_scrub_replicated() {
 EOF
 
     jq "$jqfilter" $dir/json | python -c "$sortkeys" | sed -e "$sedfilter" > $dir/csjson
-    diff -y $termwidth $dir/checkcsjson $dir/csjson || test $getjson = "yes" || return 1
+    diff ${DIFFCOLOPTS} $dir/checkcsjson $dir/csjson || test $getjson = "yes" || return 1
```

Comment:
@tchaikov 
I see ...
If you feel strong about this we can do this, but just keep diff as regular external command has my preference. Overiding names like this can always lead to obfuscation, even it needs a `$` to catually call the scripted command. So at least I'd like to use a distinctive name.